listen_to_mqtt.sh

```diff
@@ -61,8 +61,9 @@ listen_to_mqtt() {
           ;;
 
         scan-bleln-macaddr)
-          log_notice "Scanning for Tesla BLE Local Name and respective MAC addr..."
-          scan-bleln-macaddr
+          ble_ln=$(tesla_vin2ble_ln $vin)
```

Comment:
you don't need this here, already calculated in `scan-bleln-macaddr`

Reply:
I moved the calculation from the scan-bleln-macaddr to here; 1) makes that function shorter 2) if someday we want to use that function elsewhere in the code, we have access to the ble_ln everywhere... that actually makes me think, with the vin we can possibly get the ble_ln instead of calculating it.

Lets leave it like this for now, anyhow MAC address presence are going away in the next BLE Presence revamp.

Reply:
Oops you are right, it was still there but I had removed it earlier. When I merged probably didn't pay attention when resolving a conflict.